.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+*.code-workspace
```

Comment:
Looks great. However, we could add other VSCode files too. WDYT? Here's a link: https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore

Reply:
Done.